auth/src/main/java/com/firebase/ui/auth/ui/idp/SingleSignInActivity.java

```diff
@@ -90,6 +93,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         mProvider.getOperation().observe(this, new ResourceObserver<IdpResponse>(this) {
             @Override
             protected void onSuccess(@NonNull IdpResponse response) {
+                // TODO: Audit this use of SOCIAL_PROVIDERS
```

Comment:
TODO(me): Do this

Reply:
Bump - not sure what this means, but apparently you should do it :P